projects/plugins/crm/.gitattributes

```diff
@@ -1,24 +1,7 @@
-# Files not needed to be distributed in the package.
-.gitattributes       export-ignore
-.github/             export-ignore
-package.json         export-ignore
-gulpfile.esm.js      export-ignore
-.env                 export-ignore
-.env.testing         export-ignore
-codeception.yml      export-ignore
-codeception.dist.yml export-ignore
-.wp-env.json         export-ignore
-/.wordpress-org/**/* export-ignore
-/.wordpress-org      export-ignore
-.wordpress-org/**/*  export-ignore
-.wordpress-org       export-ignore
```

Comment:
I assume this won't impact the ability for this folder to control the WP.org repo assets? Or if it does, how are said assets updated going forward?

Reply:
Looking into it, it (`export-ignore`) is related to packages (I don't see a Jetpack CRM Packagist entry for CRM), and the `export-ignore` meant for building an archive.

As for `production-exclude`, and `production-include`, they're focussed on what is being built based on what is already in the monorepo. For example `.gitignore` makes sure we don't commit changes to the 'build' directory, but when we're creating a release and the release is added to the mirror repo, in the process we're running the production build commands and the build directory is built. We do want that in production (just not in the monorepo), so we `production-include` it. Same for `production-exclude`, but it's for things we want in the monorepo and not in the mirror repo / production.

I am not too sure how CRM use `export-ignore` previously in `.gitattributes` - that's all there were in there. Likely in the way the plugin was built / packaged. Now we're using `production-include` and `production-exclude`. 

I've also been using the other standalones and their `.gitattributes` as guides ([Social](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/.gitattributes), [Search](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/search/.gitattributes) for example). For example we previously included node-modules as an export-ignore, however this is now not included in `.gitattributes` (just in .`gitignore`), also in line with the other standalones.

As for `.wordpress-org`, in 5.6.0 that was in both the mirror repo and the download on SVN. Checking 5.5.3 I see it wasn't available in the download, however I'm also not too sure how that is used to build the assets so  I'm not too sure about adding that to `production-exclude`. It was in the `.distignore` for 5.6.0 as well but still displays in the plugin download.